docs/config/supernode_properties.md

```diff
@@ -0,0 +1,72 @@
+# Customize supernode properties
+
+This topic explains how to customize the dragonfly supernode startup parameters.
+
+## Parameter instructions
+
+### The parameters we can configure in supernode is as follows
+
+The following startup parameters are supported for `supernode`
+
+| Parameter | Default | Description |
+| ------------- | ------------- | ------------- |
+| listenPort | 8002 | listenPort is the port that supernode server listens on |
+| downloadPort | 8001 | downloadPort is the port for download files from supernode |
+| homeDir | /home/admin/supernode | homeDir is the working directory of supernode |
+| advertiseIP | localhost | the supernode ip is the ip we advertise to other peers in the p2p-network |
+| schedulerCorePoolSize | 10 | pool size is the core pool size of ScheduledExecutorService |
+| downloadPath | /home/admin/supernode/repo/download | DownloadPath specifies the path where to store downloaded files from source address |
+| peerUpLimit | 5 | upload limit for a peer to serve download tasks |
+| peerDownLimit | 4 |the task upload limit of a peer when dfget starts to play a role of peer |
+| eliminationLimit | 5 | if a dfget fails to provide service for other peers up to eliminationLimit, it will be isolated |
+| failureCountLimit | 5 | when dfget client fails to finish distribution task up to failureCountLimit, supernode will add it to blacklist|
+| linkLimit | 20480GB | LinkLimit is set for supernode to limit every piece download network speed |
+| systemReservedBandwidth | 20480GB |  network rate reserved for system |
+| maxBandwidth | 204800GB | network rate that supernode can use |
+| enableProfiler | false | profiler sets whether supernode HTTP server setups profiler |
+| debug | true | switch daemon log level to DEBUG mode |
+| failAccessInterval | 3m0s | fail access interval is the interval time after failed to access the URL |
+| gcInitialDelay | 6s | gc initial delay is the delay time from the start to the first GC execution |
+| gcMetaInterval | 2m0s | gc meta interval is the interval time to execute the GC meta |
+| taskExpireTime | 3m0s | task expire time is the time that a task is treated expired if the task is not accessed within the time |
+| peerGCDelay | 3m0s | peer gc delay is the delay time to execute the GC after the peer has reported the offline |
+| gcDiskInterval | 15s | GCDiskInterval is the interval time to execute GC disk |
+| youngGCThreshold | 100GB | if the available disk space is more than YoungGCThreshold and there is no need to GC disk |
+| fullGCThreshold | 5GB | if the available disk space is less than FullGCThreshold and the supernode should gc all task files which are not being used |
+| IntervalThreshold | 2h0m0s | IntervalThreshold is the threshold of the interval at which the task file is accessed |
+| cleanratio | 1 | CleanRatio is the ratio to clean the disk and it is based on 10 |
```

Comment:
Actually, the `cleanratio` can not be parsed normally. Could you please update it BTY?

Reply:
> Actually, the `cleanratio` can not be parsed normally. Could you please update it BTY?

so do you mean add  the yaml in config.go as follows
`// CleanRatio is the ratio to clean the disk and it is based on 10.
	// It means the value of CleanRatio should be [1-10].
	//
	// default: 1
	CleanRatio int
`
or delete this parameter in this doc

Reply:
IMOP, delete it for now .

Reply:
ok

Reply:
Done